LockAdvancedTest Failures

```
java.lang.AssertionError: tryLock() is not interrupted!
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.assertTrue(Assert.java:41)
    at com.hazelcast.concurrent.lock.LockAdvancedTest.testLockInterruption(LockAdvancedTest.java:226)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-OracleJDK8/com.hazelcast$hazelcast/886/testReport/junit/com.hazelcast.concurrent.lock/LockAdvancedTest/testLockInterruption/

**testLockInterruption**
```
java.lang.AssertionError: tryLock() is not interrupted!
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.concurrent.lock.LockAdvancedTest.testLockInterruption(LockAdvancedTest.java:226)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast/1090/testReport/junit/com.hazelcast.concurrent.lock/LockAdvancedTest/testLockInterruption/

**testLockInterruptibly**
```
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.hazelcast.concurrent.lock.LockAdvancedTest.testLockInterruptibly(LockAdvancedTest.java:391)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast/1090/testReport/junit/com.hazelcast.concurrent.lock/LockAdvancedTest/testLockInterruptibly/

--- a/hazelcast/src/test/java/com/hazelcast/concurrent/lock/LockAdvancedTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/concurrent/lock/LockAdvancedTest.java
@@ -121,7 +121,7 @@ public class LockAdvancedTest extends HazelcastTestSupport {
             }
         });
         t.start();
-        assertTrue(latch.await(30, TimeUnit.SECONDS));
+        assertOpenEventually(latch, 30);
     }
 
 
@@ -166,7 +166,7 @@ public class LockAdvancedTest extends HazelcastTestSupport {
         latch2.await(3, TimeUnit.SECONDS);
         Thread.sleep(500);
         lock.unlock();
-        assertTrue(latch.await(5, TimeUnit.SECONDS));
+        assertOpenEventually(latch, 5);
     }
 
     //todo:   what does isLocked2 test?
@@ -223,7 +223,7 @@ public class LockAdvancedTest extends HazelcastTestSupport {
         t.start();
         Thread.sleep(2000);
         t.interrupt();
-        assertTrue("tryLock() is not interrupted!", latch.await(30, TimeUnit.SECONDS));
+        assertOpenEventually("tryLock() is not interrupted!", latch, 30);
         lock.unlock();
         assertTrue("Could not acquire lock!", lock.tryLock());
     }
@@ -252,7 +252,7 @@ public class LockAdvancedTest extends HazelcastTestSupport {
         });
         t.start();
         lockOwner.shutdown();
-        assertTrue(latch.await(10, TimeUnit.SECONDS));
+        assertOpenEventually(latch, 10);
     }
 
     @Test(timeout = 100000)
@@ -278,7 +278,7 @@ public class LockAdvancedTest extends HazelcastTestSupport {
         });
         t.start();
         lockOwner.shutdown();
-        assertTrue(latch.await(10, TimeUnit.SECONDS));
+        assertOpenEventually(latch, 10);
     }
 
     private void assertAllLocked(ILock... locks) {
@@ -329,7 +329,7 @@ public class LockAdvancedTest extends HazelcastTestSupport {
         assertTrue(lock1.tryLock());
         lock1.unlock();
         lock1.unlock();
-        assertTrue(latch.await(10, TimeUnit.SECONDS));
+        assertOpenEventually(latch, 10);
     }
 
     @Test(timeout = 100000)
@@ -388,7 +388,7 @@ public class LockAdvancedTest extends HazelcastTestSupport {
         t.start();
         sleepMillis(5000);
         t.interrupt();
-        assertTrue(latch.await(30, TimeUnit.SECONDS));
+        assertOpenEventually(latch, 30);
     }
 
     @Test

--- a/hazelcast/src/test/java/com/hazelcast/concurrent/lock/LockAdvancedTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/concurrent/lock/LockAdvancedTest.java
@@ -388,7 +388,7 @@ public class LockAdvancedTest extends HazelcastTestSupport {
         t.start();
         sleepMillis(5000);
         t.interrupt();
-        assertTrue(latch.await(15, TimeUnit.SECONDS));
+        assertTrue(latch.await(30, TimeUnit.SECONDS));
     }
 
     @Test

